zerver/lib/management.py

```diff
@@ -48,6 +73,33 @@ def get_realm(self, options):
             raise CommandError("There is no realm with id '%s'. Aborting." %
                                (options["realm_id"],))
 
+    def get_users(self, options, realm):
+        # type: (Dict[str, Any], Optional[Realm]) -> List[UserProfile]
+        if "all_users" in options:
+            all_users = options["all_users"]
+
+            # User should pass either user list or all_users flag
+            if bool(options["users"]) == all_users:
+                self.print_help(sys.argv[0], sys.argv[1])
+                print(self.style.ERROR("Please pass either --users or --all-users."))
+                exit(1)
```

Comment:
In management commands, `raise CommandError` is the correct way to throw an exception.  It's a lot cleaner.

I think a good follow-up project might be to audit our other management commands to convert them to use `raise CommandError()`; there's not really a good reason to be calling `sys.exit` directly (and it causes problems for unit-testing the management command!).